controllers/scenario/scenario_adapter.go

```diff
@@ -58,7 +58,19 @@ func NewAdapter(application *applicationapiv1alpha1.Application, scenario *v1bet
 // in case it is, set its owner reference
 func (a *Adapter) EnsureCreatedScenarioIsValid() (controller.OperationResult, error) {
 
-	// First check if application exists or not
+	// First check if status conditions are set to prevent a known issue with incorrectly migrated v1alpha1 scenarios
+	if a.scenario.Status.Conditions == nil {
```

Comment:
Are we sure that currently-present-scenarios with this issue, at least have the `.status` field defined?

Reply:
Yes, status is a subresource so it always gets created, but it's empty for migrated resources.
Here's an example from the staging cluster:
![image](https://github.com/redhat-appstudio/integration-service/assets/34320458/5ee289cf-12cb-4421-9f03-6c588fe49ad4)

Reply:
Thanks for the explanation and screenshot!